include/seqan3/alphabet/gap/gapped.hpp

```diff
@@ -53,8 +53,6 @@ namespace seqan3
  *
  * The gapped alphabet represents the union of a given alphabet and the
  * seqan3::gap alphabet (e.g. the four letter DNA alphabet + a gap character).
- * Note that you cannot assign regular characters, but additional functions for
- * this are available.
```

Comment:
I spoke with @eseiler said that he removed this here, because this class has no own `::A, ::C` members.

This is fine with me.

Reply:
I think the gapped alphabet has a static member `gap::GAP` and can be assigned from char `-`. So I guess this cannot be removed but needs to be replaced ?